### PR TITLE
all

### PR DESCRIPTION
**Pull request by Martno96**
my pull request :)

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -56,6 +63,13 @@ <h1>Does the person have</h1>
           alt="Guess Who"
         />
         <h1 id="winOrLoseText"></h1>
+        <div class="game-board" id="secret-person-reveal">bjeb</div>
+        <!-- <img
+          id="secret-image"
+          class="secret-image"
+          src="images/guess-who-bubble.png"
+          alt="Secret"
+        /> -->
```
Comment (MariaPogosjan) [inline code/index.html:72]: A small thing but maybe delete what you have commented out and don't want to use.

File: code/index.html [hunk 2]
```diff
@@ -25,16 +25,23 @@ <h1>Does the person have</h1>
       <select id="questions">
         <optgroup label="hair color">
```
Comment (MariaPogosjan) [inline code/index.html:26]: In the <head>, in title change to _Guess who_ so it look good on your browser tab.

File: code/style.css [hunk 3]
```diff
@@ -177,6 +182,23 @@ button {
...
+}
+
+.win-or-lose .card:hover img,
+.win-or-lose .card:hover p {
+  display: flex;
+}
+
 @media (max-width: 768px) {
   body {
     flex-direction: column;
```
Comment (MariaPogosjan) [inline code/style.css:204]: In @media - .game-board, maybe add justify-content and center the images so it looks the same in mobile screen

File: code/script.js [hunk 4]
```diff
@@ -1,7 +1,30 @@
...
+//    [X] reconsider creating a new property for currentQuestion, but refresh yourself on why making an if statement or just NOT importing the category at all might be easier :P consider an alternative source for the text than .category which may already exist :))
+//[X] in filterCharacters, move out the filter method calls from each category message ifs and elses and put them in a single separate if else at the end
+//[_] properly implement the default choice of "choose here" (I think the html is what is left) also credit Maria and the other girl (Maria pinged us?)
+//[_] either keep labels or remove them and find a way to make the category names prettier instead
+ 
+//Extra flare:
+//[X] Add portrait of the secret person on the winOrLose screen
+//[_] Make my friends and memes as characters in a theme overhaul (4 h)
+
+
```
Comment (MariaPogosjan) [inline code/script.js:18]: I love that you did this to-do in your js. I will do the same next time so I know what I have and haven't done. Great idea 🙌🏻

File: code/script.js [hunk 5]
```diff
@@ -251,93 +274,168 @@ const setSecret = () => {
...
+      label: 'eyes'
+    }
+  } else if (category === 'accessories') {     
+    currentQuestion = {
+      attribute: questions.options[questions.selectedIndex].value, //reaches into the selected option tag itself to get the right subcategory of accessory (hat or glasses)
+      value: true,
       category: category,
+      label: questions.options[questions.selectedIndex].textContent //fetches the literal text of the option selected, which is exactly what the user sees
     }
+
```
Comment (MariaPogosjan) [inline code/script.js:311]: Impressive to solve this in this way. Didn't know that!  🏅

File: code/script.js [hunk 6]
```diff
@@ -251,93 +274,168 @@ const setSecret = () => {
...
 }
 
 // This function should be invoked when you click on 'Find Out'.
 const checkQuestion = () => {
-  // Compare the currentQuestion with the secret person.
-  // See if we should keep or remove people based on that
-  // Then invoke filterCharacters
+
+  console.log(`you chose ${currentQuestion.value} ${currentQuestion.category} `)
+  console.log(`the secret person has ${secret[currentQuestion.attribute]} ${currentQuestion.category}`) //
```
Comment (MariaPogosjan) [inline code/script.js:329]: There is some console.log left in the code.

File: code/script.js [hunk 7]
```diff
@@ -251,93 +274,168 @@ const setSecret = () => {
...
+  
+  const makeGuess = confirm(`Are you sure you want to guess on ${suspect} being the secret person?`)
+  
+  if (makeGuess) {
+
+    checkMyGuess(suspect)
+
+  } else {
+    //intentionally kept empty
+  }
```
Comment (MariaPogosjan) [inline code/script.js:401]: I don't think we need to add an else if we are not going to use it. However, good too keep it if you want to add something to it later on.

**Review by MariaPogosjan — APPROVED** [review]
Great job Martin! Everything works great and love love love that you made the character's image visible when user guesses for one of them. 🏆

I added some comments to make the layout look better in the mobile screen and cleaning the code.

File: code/index.html (hunk 1 above)
Comment (KarolCislowski) [reply code/index.html:72]: Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️

File: code/script.js (hunk 4 above)
Comment (KarolCislowski) [reply code/script.js:18]: Yes, it is nice, but it should not be found in the final stage of the project.

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**